doc/Repeater.md

```diff
@@ -1,8 +1,63 @@
+# Quantum Repeater
 
-*Note: this file is an early design document, but might be updated as the version goes up.*
+## What is this?
 
-# Repeater #
+Quantum Repeater is one crucial network device in entanglement-based quantum networking, which allows us to extend link-level entanglement to end-to-end entanglement. 
 
-A Repeater node has exactly two external links, so that it is useful in a line only.  It is essentially a repurposed [QNIC](QNIC), with both halves of the card running the same [Network Type](Network Type). It also contains a simple [C4](C4).
+The main works of quantum repeaters are following.
```

Comment:
Here, in fact, the lines describing the Repeater implementation as having two interfaces has been deleted!

Reply:
OK, let me add it again.